doc/reference/modules/basic_mapping.xml

```diff
@@ -3982,12 +3982,11 @@
             </para>
 
             <para>
-                The <literal>ICompositeUserType</literal>, <literal>IEnhancedUserType</literal>,
```

Comment:
I have removed the mention of `ICompositeUserType` since it is already mentioned in the previous paragraph, along with `IUserType`.